posix.mak

```diff
@@ -59,7 +59,8 @@ DOC_TOOLS = \
     $(ROOT)/dman
 
 TEST_TOOLS = \
-    $(ROOT)/rdmd_test
+    $(ROOT)/rdmd_test \
+    $(ROOT)/tests_extractor
```

Comment:
`tests_extractor` cannot be built with just DMD, so this isn't right.

Reply:
Can you explain more what you see as the problem here?  From my side the point is simply that `clean` should remove all generated files (and this ensures that).

Reply:
See the target using `TEST_TOOLS` below:

```
 $(TOOLS) $(DOC_TOOLS) $(CURL_TOOLS) $(TEST_TOOLS): $(ROOT)/%: %.d
$(DMD) $(DFLAGS) -of$(@) $(<)
```

Reply:
Ouch!  Must say that it's weird then that the CI doesn't catch this.  Is the explicit `$(ROOT)/tests_extractor` target being chosen in preference to the wildcard-y one?

Reply:
No idea, actually.

Reply:
I realized I missed a warning output by `make` about the shadowing of the target definition.  Fixing.